CorsixTH/com.corsixth.corsixth.metainfo.xml

```diff
@@ -20,6 +20,90 @@
+          <li>For games compiled using fluidsynth (such as our Windows and Flatpak
+              builds), you can now set the soundfont in the configuration file</li>
+          <li>Video files can now be missing during TH installation corruption
+              checks</li>
+        </ul>
+        <p>Translations</p>
+        <ul>
+          <li>The original game&apos;s numerals for the money bar can now be
+              toggled by translators using the isArabicNumerals flag. This setting
+              is set to off currently for CJK languages</li>
+          <li>Reworded cheat failure messages, some translations still require
+              updating</li>
+          <li>Improved hotkey window and research tooltips, some translations
+              still require updating</li>
+          <li>A new Japanese translation has been added. Thanks @ShimaKen523</li>
+          <li>Brazilian Portuguese translation has been updated. Thanks
+              @altiereslima</li>
+          <li>Russian translation has been updated. Thanks @ARGAMX</li>
+          <li>French translation has been updated. Thanks @Sanndow</li>
+          <li>Dutch translation has been updated. Thanks @Alberth289346 &amp;
+              @jetenergy</li>
+        </ul>
+        <p>Bug Fixes</p>
+        <ul>
+          <li>Game speed tick rates have been smoothed out again, improving
+              animations and sound playback for objects</li>
+          <li>Improved colour accuracy for fonts in unicode languages now means
+              you can actually read when your game is paused</li>
+          <li>Many fixes for the position of staff and patient moods in the game!
+              More are scheduled to come in future</li>
+          <li>Fixed a (very annoying) bug with walks that caused new patients to
+              enter rooms along their path</li>
+          <li>Rework to the operating theatre room to prevent patient/staff
+              lockups</li>
+          <li>Free build now always comes with free heating and free parcel
+              purchases!</li>
+          <li>Epidemic timer now ends once all patients are resolved</li>
+          <li>Old serious radiation epidemics can no longer cause a game crash.
+              In future, this epidemic type will be reinstated again</li>
+          <li>Failed cheats will now always provide feedback messages</li>
+          <li>Spawning a contagious patient via cheating now works in levels
+              without a helipad</li>
+          <li>Long windows now correctly count as a window</li>
+          <li>Dynamic info for machines now displays correctly when cycling between
+              them</li>
+          <li>Fixed a bug where changing doctor in General Diagnosis could break
+              the room and patient</li>
+          <li>Fixed a scenario where clicking on a staff member in an exploding
+              room caused a game crash</li>
+          <li>And more... Check out the full project changelog for the full list</li>
```

Comment:
Was meant to be 'check out the project changelog'. Fix after if possible if should be merged now